.registry/app.yaml

```diff
@@ -1,4 +1,4 @@
-title: "Minimal GCP Environment"
+title: "GCP Sample Environment"
```

Comment:
Should this be `GCP Sample Stack` or do we want to keep `Environment` here and in other files?

Reply:
Not sure, I didn't change it because we were using "Environment" even though this is a "Stack" already. I think it was because that is kind of the purpose of this stack, but I would be happy to update 👍 @prasek what do you think?

Reply:
I would use "GCP Sample Stack" like we do for "GCP Provider" even though that's a provider already.